docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,130 @@
+---
+title: MUI X improvements, Charts in Alpha, Discord, and more!
```

Comment:
I believe Discord launch deserves its own launch, we could invite users to this blog post, but I wouldn't have it on the title.

```suggestion
title: MUI X improvements, Charts in Alpha, Discord, and more!
```

Reply:
Also, I think we should retain the reference to v6.10.0 in the title.

Reply:
I was about to leave the same comments, great

Reply:
removed for now. Will do it's own blog